packages/block-library/src/table/editor.scss

```diff
@@ -3,9 +3,14 @@
 	&[data-align="right"],
 	&[data-align="center"] {
 		table {
+			display: table;
```

Comment:
I would like for us to get @ellatrix input on this one before we ship. I'm almost certain that I've explored something similar back in the past in efforts to make the table responsive, but every solution I found had some breaking downsides.

Reply:
I forgot the details as well. Isn't `table` the default `display` value in case of a `table` element? Why only for aligned tables? This should receive an inline comment.

Reply:
Yup, sorry, I should have described the changes better. I'm setting the default display for the table to `block` so that, with `overflow-x: scroll`, it can scroll horizontally on small breakpoints, as recommended in the issue. However, for fixed-width tables and for aligned tables that won't work, so I'm resetting the display value to `table` for those. I'll put a comment in to make it clear.

Reply:
Done!

Reply:
For future reference: it is important to note that [some CSS display properties do affect table semantics](http://adrianroselli.com/2018/02/tables-css-display-properties-and-aria.html#TheTable). For example, flex | block | grid | inline, they all override the table native semantics, rendering it useless to a screen reader. Keeping the native display: table is the right way to go for data tables, thanks for that! 🙂